src/daemon/bitcoind/actions.rs

```diff
@@ -614,7 +757,7 @@ fn unconfirm_vault(
     }
 
     match vault.status {
-        VaultStatus::Unconfirmed => unreachable!(),
+        VaultStatus::Unconfirmed => unreachable!("Unconfirming a confirmed vault"),
```

Comment:
nit: you meant "an unconfirmed vault"?

Reply:
i mean we are unconfirming a confirm vault, as stated in the doc comment

Reply:
Yeah maybe not much clear but meh